src/test/java/com.epam.rp/api/AllureSampleTest.java

```diff
@@ -4,10 +4,10 @@
 import io.qameta.allure.Feature;
 import org.junit.jupiter.api.Test;
 
-@Feature("Login")
+@Feature("SampleFeature")
 public class AllureSampleTest {
     @Test
     public void testUserLogin() {
-        Logger.logDebugMessage("ALLURE debug test message");
+        Logger.logDebugMessage("ALLURE debug API test message");
```

Comment:
The idea o API part of the task was in creating one simple, but real API test. Please prosede with it in scope of one of nearest modules.

Reply:
Sure, will do it in my next PR